### PR TITLE
Make stub source icon part of `SourceIcon`

### PR DESCRIPTION
**Pull request by AntsyLich**

### CODE REVIEW

File: app/src/main/java/eu/kanade/presentation/browse/components/BrowseIcons.kt [hunk 1]
```diff
@@ -37,21 +39,33 @@ private val defaultModifier = Modifier
...
-            modifier = modifier.then(defaultModifier),
-        )
-    } else {
-        Image(
-            painter = painterResource(id = R.mipmap.ic_local_source),
-            contentDescription = "",
-            modifier = modifier.then(defaultModifier),
-        )
+    when {
+        source.isStub && showStubSourceIcon -> {
```
Comment (ghostbear) [inline app/src/main/java/eu/kanade/presentation/browse/components/BrowseIcons.kt:47]:
```suggestion
        source.isStub && icon == null-> {
```

File: app/src/main/java/eu/kanade/presentation/browse/components/BrowseIcons.kt [hunk 2]
```diff
@@ -37,21 +39,33 @@ private val defaultModifier = Modifier
...
+            Image(
+                painter = painterResource(R.drawable.ic_warning_white_24dp),
```
Comment (ghostbear) [inline app/src/main/java/eu/kanade/presentation/browse/components/BrowseIcons.kt:49]: Maybe add a new icon to match the others

**Review by arkon — COMMENTED** [review]
Should be able to update `MigrateSourceItem` as well.

**Comment by arkon** [comment]
nvm I'm dumb

**Pull request merged**